app/controllers/manage_proposal_controller.rb

```diff
@@ -70,4 +91,42 @@ def user_has_approver_permissions?
   def unauthorized?(response)
     response.status == 401
   end
+
+  def publish_delete_proposal(proposal, provider)
```

Comment:
I'm not saying this is very different, but we should try and align our logging and flash messages. Since this is most like deleting a record, I think we should align the logging and flash with https://github.com/nasa/mmt/blob/1d0064d9492f56be0bd03485b49b6deedc7814a1/app/controllers/collections_controller.rb#L38 and https://github.com/nasa/mmt/blob/1d0064d9492f56be0bd03485b49b6deedc7814a1/app/controllers/base_published_record_controller.rb#L67. I think there are just some minor changes that need to be made.

Reply:
will changes be made?

Reply:
They were already made

Reply:
Revisiting the flashing, I think that they are already aligned with other flashes.  The general error messages have the same format.  The granules error is the same message we display when the user tries to delete a collection already (and has been approved by Alicia).  The not found error was not approved.  I can get it approved when she returns, if the wording seems problematic.  I don't think there is a need to hold this ticket up for that since MMT-1962 is related and can adjust the wording, if necessary.